lms/static/js/dashboard/views/course_list_view.js

```diff
@@ -0,0 +1,39 @@
+;(function (define) {
+    'use strict';
+
+    define(['jquery', 'underscore', 'common/js/components/views/list', 'text!templates/dashboard/courses.underscore'],
+        function ($, _, ListView, coursesTemplate) {
+
+            var CourseListView = ListView.extend({
+
+                initialize: function (options) {
+                    this.itemViewClass = options.itemViewClass || this.itemViewClass;
+                    this.template = _.template(coursesTemplate);
```

Comment:
The can be defined as a parameter for the view instead of in the initialize function, e.g.
``` template: _.template(courseTemplate),```